src/fe/implicit/_Obj.js

```diff
@@ -272,3 +272,38 @@ export const entries = o => {
 
   return list;
 };
+
+/**
+ * Returns the value at a path if value exists
+ * @param {Object} object
+ * @param {string} path
+ *
+ * @returns {*}
+ */
+export const getSafely = (object, path) => {
+  const points = path.split('.');
+  let anchor = object;
+
+  for (let i = 0; i < points.length; i++) {
+    try {
+      const item = anchor[points[i]];
+
+      // Continue only if non-primitive or string
+      if (_.isPrimitive(item) && !_.isString(item)) {
+        const isLastInPath = i === points.length - 1;
```

Comment:
Add explanation for why we're doing this.

Reply:
Added